aptos-move/aptos-release-builder/src/components/feature_flags.rs

```diff
@@ -9,9 +9,7 @@ use serde::{Deserialize, Serialize};
 
 #[derive(Clone, Deserialize, PartialEq, Eq, Serialize, Debug)]
 pub struct Features {
-    #[serde(default)]
```

Comment:
Why delete this?

Reply:
Good catch! This should be due to rebase. Fixed.